pom.xml

```diff
@@ -124,8 +128,9 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <artifactId>mockito-inline</artifactId>
       <groupId>org.mockito</groupId>
+      <artifactId>mockito-inline</artifactId>
+      <version>5.2.0</version>
```

Comment:
Can we add this to the properties, so we have a clear overview of the versions there?